### PR TITLE
Fixed webhook URLs in web UI

### PR DESCRIPTION
**Pull request by soltysh**
I've also noticed we're advertising wrong URLs in docs as well, the accompanying PR is at https://github.com/openshift/openshift-docs/pull/463

### CODE REVIEW

**Comment by soltysh** [comment]
@liggitt @smarterclayton PTAL

File: assets/app/scripts/services/data.js [hunk 1]
```diff
@@ -830,7 +830,7 @@ angular.module('openshiftConsole')
       template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
     }
     else if (id) {
-      if (type === "buildConfigHooks") {
+      if (type === "buildconfighooks") {
```
Comment (liggitt) [inline assets/app/scripts/services/data.js]: in DataService.prototype.url, we also have to normalize type:
```
var type = normalizeType(options.type);
```

Comment (soltysh) [inline assets/app/scripts/services/data.js]: Done.

File: assets/app/scripts/services/data.js [hunk 2]
```diff
@@ -860,7 +860,8 @@ angular.module('openshiftConsole')
       var opts = angular.copy(options);
       delete opts.type;
       delete opts.id;
-      return this._urlForType(options.type, options.id, null, false, opts).toString();
+      type = normalizeType(options.type)
```
Comment (liggitt) [inline assets/app/scripts/services/data.js]: `var type`

Comment (liggitt) [inline assets/app/scripts/services/data.js]: And end the line with `;`

Comment (soltysh) [inline assets/app/scripts/services/data.js]: Done.

**Comment by smarterclayton** [comment]
Approved

**Comment by liggitt** [comment]
[merge]

**Comment by jhadvig** [comment]
@soltysh :+1:

**Pull request merged**